### PR TITLE
AIRNEIS-40 dive in update Makefile

### PR DESCRIPTION
**Pull request by BarnabePILLIAUDIN**
DIve in for the update of Makefile.

### CODE REVIEW

File: docs/dive-in/2024/01/2024-01-16:update-make-file.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Make
+
+Make is a utility to automate somme commands.  
+Also it allows to have a kind of consistency between commands or to simplify commands.  
+It allows with a single command to run that will run multiple commands.
```
Comment (Magali-D) [inline docs/dive-in/2024/01/2024-01-16:update-make-file.md]: Sentence unclear, can you reformulate?

File: docs/dive-in/2024/01/2024-01-16:update-make-file.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Those commands are registered in the `Makefile` and have the same skeleton:  
+`make [commandName] [arguments]`
```
Comment (cedricgautier) [inline docs/dive-in/2024/01/2024-01-16:update-make-file.md:9]: What kind of arguments will you put ?

Comment (barnabepilliaudin) [reply docs/dive-in/2024/01/2024-01-16:update-make-file.md:9]: For the commands of the tasks there won't be any arguments

File: docs/dive-in/2024/01/2024-01-16:update-make-file.md (hunk 1 above)
Comment (barnabepilliaudin) [reply docs/dive-in/2024/01/2024-01-16:update-make-file.md]: You can with a single command execute multiple command 

for exemple make start-then-stop:
 could execute docker compose up then docker compose down (even though this command would be the definition of useless :) )

Comment (barnabepilliaudin) [reply docs/dive-in/2024/01/2024-01-16:update-make-file.md]: Pushed a new version with a better exemple (wasn't too difficult 😂)

**Review by Magali-D — APPROVED** [review]

**Review by cedricgautier — APPROVED** [review]

**Pull request merged**